src/category_theory/limits/limits.lean

```diff
@@ -582,10 +662,25 @@ are themselves isomorphic.
 @[simps]
 def cocone_points_iso_of_nat_iso {F G : J ⥤ C} {s : cocone F} {t : cocone G}
   (P : is_colimit s) (Q : is_colimit t) (w : F ≅ G) : s.X ≅ t.X :=
-{ hom := P.desc ((limits.cocones.precompose w.hom).obj t),
-  inv := Q.desc ((limits.cocones.precompose w.inv).obj s),
-  hom_inv_id' := by { apply hom_ext P, tidy, },
-  inv_hom_id' := by { apply hom_ext Q, tidy, }, }
+{ hom := P.map t w.hom,
+  inv := Q.map s w.inv,
+  hom_inv_id' := P.hom_ext (by tidy),
+  inv_hom_id' := Q.hom_ext (by tidy) }
+
+lemma comp_cocone_points_iso_of_nat_iso_hom {F G : J ⥤ C} {s : cocone F} {t : cocone G}
```

Comment:
Should these also be reassoc'ed?

Reply:
Actually, I'm not altogether clear why we even have this lemma and the next one. `simp` will do them already, and there doesn't appear to be any immediate need for them as `rw` lemmas.

Reply:
@jcommelin They should, thanks!

@semorrison Yeah I see your point, but I think I added these so that things like `has_colimit.iso_of_nat_iso_ι_hom` can happen without unfolding definitions (and a couple more examples in the preserves limits PR I'm making). If I remember correctly, `simp` struggled to do the versions of these where one of the colimits was the one chosen using `has_colimit` but having these intermediate ones worked as `rw` lemmas instead.

Reply:
Yeah - looking at this again `has_colimit.iso_of_nat_iso_ι_hom` was already there as a simp lemma, and I think it's a good idea for everything with a `has_` version to have a corresponding `is_` version.